index.html

```diff
@@ -43,7 +43,9 @@
 
     /* ------ POSITION GAME-CANVAS ------ */
     canvas#canvas {
-      width: 512px; height: 512px;                /* max game size: 512px */
+      max-width: 512px;                           /* max game size: 512px */
+      width: 512px; height: 512px;
+      width: 100vmin;                             /* ensures the game won't overflow the screen */
       image-rendering: optimizeSpeed;
```

Comment:
Great Idea to use ``vmin`` there.

I'll test this soon, but at I first look I wonder if it shouldn't be more like

```
width: 512px;
height: 512px;
max-width: 100vmin;
max-height: 100vmin;
```

Reply:
I went for the ``max-width`` in px and ``width`` in vmin in order to keep the change as small as possible. In practice, both are interchangeable without any change in behavior – the canvas will still cap out at either 512px or 100vmin. Your suggestion feels a lot clearer, though.